### DISCUSSION TITLE
Funcionalidade de esconder menu lateral

### DISCUSSION THREAD

**Discussion by eliumoraes** [balta-io/7194#19] [Perguntas e Respostas] [answered]
Boa noite Balta!

Recentemente fiquei sem um segundo monitor e estou colocando o navegador em parelelo com o vscode, isso me fez enxergar que o menu lateral é totalmente incomodo ao assistir as aulas.

Estou usando uma extensão no Chrome, a Amino, e faço um display: none via css no seu menu, mas seria muito mais amigável se tu pudesse colocar a funcionalidade de menu "hamburguer", que reduz dependendo do tamanho da tela.

Tenho também cursos na udemy e eles utilizam essa funcionalidade, que ajuda muito ao colocar paralelamente dois programas.

Olha só que gostoso fica com o menu oculto:

![image](https://user-images.githubusercontent.com/19482828/131764768-4e4a998c-9bbb-4c14-80cf-5a830c30936a.png)

E o desespero com ele ocupando espaço:
![image](https://user-images.githubusercontent.com/19482828/131764836-d671cca6-afb6-4472-a1bf-24d4a0af8cf7.png)

**Comment by andrebaltieri** [comment, marked as answer]
Obrigado pelo feedback @eliumoraes
